hooks/useProposals.ts

```diff
@@ -211,7 +229,7 @@ const useL2ProposalsState = (l2Proposals?: { proposalId: string; startBlock: str
     data: l2Proposals?.map((proposal, i) => {
       return {
         proposalId: proposal.proposalId,
-        state: proposalState?.[i].result,
+        state: proposalState?.[i].result === 6 ? 'closed' : proposalState?.[i].result,
```

Comment:
works for me; i might suggest a small comment here explaining why `6` needs to be special cased

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/154/commits/2f2c309eb14c80b88ac5a6a31b3f57795612f6de